src/third_party/catch2/CMakeLists.txt

```diff
@@ -1,20 +1,17 @@
-# For more info on interface libraries,
-# see https://cmake.org/cmake/help/v3.10/command/add_library.html#object-libraries
-add_library(catch2_headers INTERFACE)
-target_include_directories(catch2_headers
-    INTERFACE
-        $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/include>
-        $<INSTALL_INTERFACE:include>
-)
 
-# For more info on object libraries,
-# see https://cmake.org/cmake/help/v3.10/command/add_library.html#object-libraries
-add_library(catch2 OBJECT ${CMAKE_CURRENT_SOURCE_DIR}/main.cpp)
+add_library(catch2 SHARED ${CMAKE_CURRENT_SOURCE_DIR}/main.cpp)
 target_include_directories(catch2
-    PRIVATE
+    PUBLIC
         $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/include>
         $<INSTALL_INTERFACE:include>
 )
 
+install(TARGETS  catch2
```

Comment:
this..may be "wrong" because it means we export catch2 but this made things a lot easier

Reply:
So maybe now's the time to bring up that I don't believe this `GennyLibraryConfig` is bringing very much value at the moment. It would be reasonable if people were developing actors outside of genny, but that's not the case and I doubt it will be for some time to come.

Reply:
Ya good call, I killed `GennyLibraryConfig`